debian/control

```diff
@@ -0,0 +1,19 @@
+Source: souffle
+Maintainer: Nathan Keynes <nathan.keynes@oracle.com>
+Section: misc
+Priority: optional
+Standards-Version: 3.9.5
+Build-Depends: autoconf, automake, bison, debhelper (>= 9), flex, libsqlite3-dev, openjdk-7-jdk | openjdk-8-jdk
+Homepage: https://github.com/oracle/souffle
+Vcs-Git: https://github.com/oracle/souffle.git
+
+Package: souffle
+Architecture: any
+Depends: ${shlibs:Depends}, ${misc:Depends}, g++, jarwrapper, openjdk-7-jre | openjdk-8-jre
```

Comment:
I do not suppose it is very difficult but it is a bit of work. We will need to get the java files out, repack and add an extra folder with deb specs. Then we need to make profiler to be somewhat independent (or specify commands for debhelper so it knows how to build it).  Also to do it properly I'd suggest that we fix configure to compile things conditionally. At the moment configuration will fail if java is not found. Finally I need to look into how to specify dependencies between debs, I suppose that profiler should depend on souffle.  If not critical, may I suggest that we leave this change for later? I can look into this (and for Bernhard's fork also).